packages/core/apps/core-apps-server-internal/src/core_app.ts

```diff
@@ -144,6 +146,36 @@ export class CoreAppsService {
         }
       }
     );
+
+    router.versioned
+      .put({
+        path: '/_settings',
```

Comment:
@afharo If this new route wasn't a versioned one, `access` would have inferred it to be `public` because the route path isn't prefixed with `/internal/`. It doesn't matter here (except for consistency between path strings and `access`).

Reply:
Good point! Thanks for highlighting this, @TinaHeiligers